pkg/sfu/buffer/rtpstats_sender.go

```diff
@@ -399,8 +399,12 @@ func (r *RTPStatsSender) Update(
 				"invalid start timestamp", nil,
 				"snBefore", r.extStartSN,
 				"snAfter", extSequenceNumber,
+				"snHighest", r.extHighestSN,
 				"tsBefore", r.extStartTS,
 				"tsAfter", extTimestamp,
+				"tsHighest", r.extHighestTS,
+				"firstTime", r.firstTime.String(),
+				"startTime", r.startTime.String(),
```

Comment:
This PR will hopefully fix the underlying condition of this. Will remove after checking once this gets deployed and looking good for a few days.